extensions/wikia/UserProfilePageV3/UserIdentityBoxDiscussionInfo.class.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+use Swagger\Client\ApiException;
+use Swagger\Client\Discussion\Api\ContributionApi;
+use Swagger\Client\Discussion\Api\SitesApi;
+use Wikia\DependencyInjection\Injector;
+use Wikia\Service\Swagger\ApiProvider;
+
+class UserIdentityBoxDiscussion {
+
+	const DISCUSSION_SERVICE_NAME = 'discussion';
+	const TIMEOUT = 5;
+
+	private $userId = null;
+	private $discussionActive = false;
+	private $discussionPostsCount = 0;
+	private $discussionAllPostsByUserLink = '';
+
+	function __construct($user) {
+		$this->userId = $user->getId();
+	}
+
+	public function isDiscussionActive() {
+		return $this->discussionActive;
+	}
+
+	public function getDiscussionPostsCount() {
+		return $this->discussionPostsCount;
+	}
+
+	public function getDiscussionAllPostsByUserLink() {
+		return $this->discussionAllPostsByUserLink;
+	}
+
+	public function fetchDiscussionPostsNumber() {
+		$siteId = F::app()->wg->CityId;
+
+		$this->discussionActive = $this->checkDiscussionActive($siteId);
```

Comment:
I'm seeing a number of whitespace violations in this PR. Could we update so it follows [our guidelines](https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md)?

Mostly what I'm seeing is padding need. Eg
```php
// bad
$this->discussionActive = $this->checkDiscussionActive($siteId);

// good
$this->discussionActive = $this->checkDiscussionActive( $siteId );
```